asv_bench/benchmarks/package.py

```diff
@@ -0,0 +1,22 @@
+"""
+Benchmarks for pandas at the package-level.
+"""
+import subprocess
+import sys
+
+from pandas.compat import PY37
+
+
+class TimeImport:
+    def time_import(self):
+        if PY37:
+            cmd = [sys.executable, "-X", "importtime", "-c", "import pandas as pd"]
+            p = subprocess.run(cmd, stderr=subprocess.PIPE)
+
+            line = p.stderr.splitlines()[-1]
+            field = line.split(b"|")[-2].strip()
+            total = int(field)  # microseconds
+            return total
+
+        cmd = [sys.executable, "-c", "import pandas as pd"]
+        p = subprocess.run(cmd, stderr=subprocess.PIPE)
```

Comment:
is this a lint error to assign to p?

Reply:
i think it should be, no idea why it isn't.  will fix anyway